client/js/views/GameView.vue

```diff
@@ -1217,6 +1217,7 @@ export default {
       position: absolute;
     }
     .c-deck-count {
+      margin: 6px;
```

Comment:
We generally use multiples of 8px so let's use 8 instead of 6
```suggestion
      margin: 8px;
```